### PR TITLE
fix: styling issues on layout

### PR DESCRIPTION
**Pull request by yeager-eren**
# Summary

This PR has solved two problems:
1. Zooming on input for mobile platforms (e.g Android)
2. Layout issues on responsive

Fixes RF-1177

# How did you test this change?

I changed the layout and it needs to be checked against widget.rango.exchange (next) to make sure this PR doesn't break anyyhing.

For testing zoom, you can check dev server on your local network using your mobile. Token/Blockchain selector page had the issue.

# Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Implemented a user interface (UI) change, referencing our Figma design to ensure pixel-perfect precision.

### CODE REVIEW

**Comment by RanGojo** [comment]
I saw some minor ui issues broken in this branch:

bottom border radius for home page:\
Screenshot from 2024-01-10 11-00-52


container border radius when scrolled:
this branch:
Screenshot from 2024-01-10 11-00-29
next:
Screenshot from 2024-01-10 11-00-21

**Comment by samobasquiat** [comment]
I tested it on mobile browser, web and device toolbar in web browser with Nexus to simulate keyboard. Although most of the height issues have been resolved, some issues have surfaced:
1- in the blockchain selector the scroll has been removed.
Screenshot
2- the scroll that @RanGojo mentioned before in home page, connect wallet page & history page has a problem.
Screenshot
Screenshot
3- the border radius of home page.

File: widget/embedded/src/components/Layout/Layout.styles.ts [hunk 1]
```diff
@@ -1,52 +1,44 @@
 import { darkTheme, styled } from '@rango-dev/ui';
 
+const WIDGET_HEIGHT = '700px';
 export const Container = styled('div', {
   position: 'relative',
   display: 'flex',
   flexDirection: 'column',
   borderRadius: '$primary',
-  overflow: 'hidden !important',
-  boxShadow: '15px 15px 15px 0px rgba(0, 0, 0, 0.05)',
```
Comment (samobasquiat) [inline widget/embedded/src/components/Layout/Layout.styles.ts:9]: the boxShadow has been removed. Don't we need it anymore?

File: widget/embedded/src/components/Layout/Layout.tsx [hunk 2]
```diff
@@ -52,7 +45,7 @@ function LayoutComponent(props: PropsWithChildren<PropTypes>, ref: Ref) {
   return (
     <Container
       ref={ref}
-      fixedHeight={fixedHeight}
+      auto={auto}
```
Comment (samobasquiat) [inline widget/embedded/src/components/Layout/Layout.tsx:48]: I think `auto` doesn't make sense. Can you change it to a better name? 
I don't have a better alternative name but for example `autoAdjust`, `isContentBased`, `flexibleHeight` or something.

**Review by samobasquiat — CHANGES_REQUESTED** [review]

File: widget/embedded/src/components/Layout/Layout.styles.ts (hunk 1 above)
Comment (yeager-eren) [reply widget/embedded/src/components/Layout/Layout.styles.ts:9]: I wanted to move it somewhere else but seems removed it completely by mistake.

Comment (yeager-eren) [reply widget/embedded/src/components/Layout/Layout.styles.ts:9]: fixed.

**Comment by yeager-eren** [comment]
As we talked offline, I did some changes to layout and it uses a consistent approach on how we are using layout, it should be more reliable and less bug-prone. I also hiding the rounded header on scroll.

**Review by samobasquiat — APPROVED** [review]

File: widget/embedded/src/pages/ConfirmSwapPage.tsx [hunk 3]
```diff
@@ -38,15 +38,6 @@ import { useWalletsStore } from '../store/wallets';
...
 import { getContainer } from '../utils/common';
 
-const Container = styled('div', {
-  position: 'relative',
-  width: '100%',
-
-  '& .quote-update__alert': {
-    paddingTop: '$10',
-  },
-});
```
Comment (Ikari-Shinji-re) [inline widget/embedded/src/pages/ConfirmSwapPage.tsx:48]: If we eliminate styles for specific class names, we should also remove the associated class name from the corresponding element.

File: widget/embedded/src/pages/Home.tsx [hunk 4]
```diff
@@ -28,15 +28,6 @@ import { getPriceImpact, getPriceImpactLevel } from '../utils/quote';
 import { canComputePriceImpact, getSwapButtonState } from '../utils/swap';
 import { formatBalance, isFetchingBalance } from '../utils/wallets';
 
-const Container = styled('div', {
-  display: 'flex',
-  flexDirection: 'column',
-  overflowY: 'visible',
-  '& .quote__container': {
-    paddingTop: '$2',
-  },
```
Comment (Ikari-Shinji-re) [inline widget/embedded/src/pages/Home.tsx:37]: If we eliminate styles for specific class names, we should also remove the associated class name from the corresponding element.

**Comment by Ikari-Shinji-re** [comment]
The padding in certain elements has been altered compared to the next branch.
next branch :
alert padding-top

alert padding-bottom

blockchains category padding-top
scrollable area padding-bottom

scrollable area padding-bottom


current branch:

**Comment by Ikari-Shinji-re** [comment]
The overflow is incorrect on certain pages (wallets, confirm wallets, confirm swap, swap details), leading to the disappearance of the border radius at the bottom of the page :

**Comment by Ikari-Shinji-re** [comment]
If we set a large border radius value in the widget configuration, certain pages such as wallets may experience issues:

File: widget/embedded/src/pages/Home.tsx (hunk 4 above)
Comment (yeager-eren) [reply widget/embedded/src/pages/Home.tsx:37]: fixed.

**Comment by yeager-eren** [comment]
@Ikari-Shinji-re you can review again. I did the changes.

File: widget/embedded/src/pages/ConfirmSwapPage.tsx (hunk 3 above)
Comment (yeager-eren) [reply widget/embedded/src/pages/ConfirmSwapPage.tsx:48]: fixed.

**Review by Ikari-Shinji-re — APPROVED** [review]

**Comment by RanGojo** [comment]
Please check changing width problem in different pages/states.

**Pull request merged**